include/mapbox/geojson_value_impl.hpp

```diff
@@ -0,0 +1,295 @@
+#pragma once
+
+#include <mapbox/geojson.hpp>
+#include <mapbox/geojson/value.hpp>
+
+namespace mapbox {
+namespace geojson {
+
+using error = std::runtime_error;
+
+template <typename T>
+T convert(const value &);
+
+template <>
+point convert<point>(const value &val) {
+    assert(val.is<value::array_type>());
```

Comment:
^throw too?

Reply:
added + added throw  for container of coordinates